activerecord/lib/active_record/connection_adapters/abstract_adapter.rb

```diff
@@ -279,6 +279,11 @@ def prefetch_primary_key?(table_name = nil)
         false
       end
 
+      # Does this adapter support partitioned indexes?
```

Comment:
The comment is pointless IMO.

Reply:
Yes, I think so. I just added it to keep the format, but the method name is pretty self-explanatory which makes the comment superfluous.